### PR TITLE
Caption modified

### PR DESCRIPTION
**Pull request by dhanyaramesh**
Changing caption

### CODE REVIEW

File: _posts/0000-01-04-dhanyaramesh.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![homer-agh](https://cloud.githubusercontent.com/assets/16547949/25400895/02672eca-29c2-11e7-87a5-3dc3d3110c29.jpg)
 {: .center}
 
-Caption here
+Caption here - Dhanya Ramesh
```
Comment (beardofedu) [inline _posts/0000-01-04-dhanyaramesh.md]: This image is supposed to focus on something that you really want, like a 🌯 or a 🌮 😄

File: _posts/0000-01-04-dhanyaramesh.md [hunk 2]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![homer-agh](https://cloud.githubusercontent.com/assets/16547949/25400895/02672eca-29c2-11e7-87a5-3dc3d3110c29.jpg)
```
Comment (beardofedu) [inline _posts/0000-01-04-dhanyaramesh.md]: `homer-agh` should really be renamed `homer-drool`

**Review by beardofedu — COMMENTED** [review]
Some small edits if you want to 👍

**Review by aniciu — APPROVED** [review]
Title is good

**Pull request merged**